packages/abi-gen/test-cli/fixtures/contracts/AbiGenDummy.sol

```diff
@@ -127,6 +127,8 @@ contract AbiGenDummy
     /// @return a Struct struct
     function structOutput() public pure returns(Struct memory s) {}
 
+    function methodReturningArrayOfStructs() public pure returns(Struct[] memory) {}
```

Comment:
can we make the naming more consistent somehow?

What about:
```
structOutput
structArrayOutput
```

I also like the effect that alphabetical proximity == conceptual proximity